terraso_backend/tests/graphql/mutations/test_sites.py

```diff
@@ -409,14 +409,11 @@ def transfer_site(request, user, site, project):
     return site
 
 
-@pytest.mark.parametrize(
-    "transfer_site", [("CONTRIBUTOR", "MANAGER"), ("MANAGER", "CONTRIBUTOR")], indirect=True
-)
+@pytest.mark.parametrize("transfer_site", [("CONTRIBUTOR", "MANAGER")], indirect=True)
```

Comment:
The Manager -> Contributor transfer case should be allowed based on the requirements document, so I removed that as a failure test here. (Might want someone to double-check my reasoning here though.)

Reply:
Confirmed that this is the expected behavior -- will clean up this test before merging.